manifests/kustomize/OWNERS

```diff
@@ -5,3 +5,4 @@ approvers:
 reviewers:
   - Bobgy
   - IronPan
+  - rmgogogo
```

Comment:
Just a confirmation, will you have capacity to review all of these?

Reply:
looks strange you ask this

Reply:
I don't get it. I meant you are adding yourself as reviewer for a lot of folders, that seems a lot of work to review all those PRs

Reply:
The reason is detailed in https://github.com/kubeflow/pipelines/pull/2893/files.
As tech leader and hosted owner, I need to well track the possible breaking changes, especially preloaded samples and manifest. Test (Rui's work can help a lot) also should be well take care. Our post-submit always fail now. Manifest is not well managed. We don't have sense to make the manifest change be back-compatible. Any of these would make hosted broken. Rui's test can help but test can't cover all things. Put me to reviewer list to keep an eye is a good approch. As for workload, definitelly still lever dedicated owners for a detailed review.

Reply:
@rmgogogo Thanks for the explanation! Sounds good to me.